parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ProducerManager.java

```diff
@@ -79,9 +80,7 @@ private void initProducer() {
             }
         } else {
             if (producerIsConfiguredForTransactions) {
-                throw new IllegalArgumentException("Using non-transactional producer option, but Producer has a transaction ID - " +
-                        "the Producer must not have a transaction ID for this option. This is because having such an ID forces the " +
-                        "Producer into transactional mode - i.e. you cannot use it without using transactions.");
+                throw new IllegalArgumentException("Using non-transactional producer option, but Producer has a transaction ID - " + "the Producer must not have a transaction ID for this option. This is because having such an ID forces the " + "Producer into transactional mode - i.e. you cannot use it without using transactions.");
```

Comment:
```suggestion
                throw new IllegalArgumentException("Using non-transactional producer option, but Producer has a transaction ID - " + "
                the Producer must not have a transaction ID for this option. This is because having such an ID forces the " + 
                "Producer into transactional mode - i.e. you cannot use it without using transactions.");
```